.github/workflows/validate-codeowners.yml

```diff
@@ -22,8 +24,10 @@ jobs:
         owner_checker_allow_unowned_patterns: "false"
         # GitHub access token is required only if the `owners` check is enabled
         github_access_token: "${{ secrets.REPO_ACCESS_TOKEN }}"
+
     - uses: mszostok/codeowners-validator@v0.7.1
-      if: github.event.pull_request.head.repo.full_name != github.repository
+      # This condition verifies that the PR repo does NOT equal the Github Repo and it's NOT dependabot
+      if: ${{ (github.event.pull_request.head.repo.full_name != github.repository) && (github.actor != 'dependabot[bot]') }}
```

Comment:
We shouldn't need to skip this check as it [does require any seconds](https://github.com/cloudposse/atmos/pull/454/files#diff-8e4f84c27b52c67a5b0f10df80e399d042b605b638c64e67332c74512d029b15R26) unlike the previous step.

Reply:
Addressed in https://github.com/cloudposse/atmos/pull/454/commits/d06ec637ba34c37c463fb0195e04bdabf5a410d4